src/main/java/com/epam/ta/reportportal/database/dao/TestItemRepositoryCustomImpl.java

```diff
@@ -105,6 +108,25 @@ public void updateHasChilds(String id, boolean hasChildren) {
 	}
 
 	@Override
+	public void updateItemsIssues(List<TestItem> forUpdate) {
```

Comment:
let's have the following signature:
public void updateItemsIssues(Map<ID,Issue> forUpdate) 
It will help to save time if we migrate to RDBMS